Makefile.am

```diff
@@ -28,4 +28,4 @@ libtool: $(LIBTOOL_DEPS)
 # Needed by autoconf because we use README.md instead of README.
 # See http://stackoverflow.com/q/15013672/
 README: README.md
```

Comment:
shouldn't this rule be removed and the README in dist_doc_DATA be renamed README.md?